app/views/SettingsView/styles.js

```diff
@@ -0,0 +1,54 @@
+import { StyleSheet } from 'react-native';
+
+import {
+	COLOR_SEPARATOR, COLOR_WHITE, COLOR_BORDER
+} from '../../constants/colors';
+import sharedStyles from '../Styles';
+
+export default StyleSheet.create({
```

Comment:
Do it like this:
<img src="https://user-images.githubusercontent.com/804994/58194551-35bf9080-7c9c-11e9-978f-fffc1b89fd53.png" width="350" />


Look at the borders, item height, margins, backgroundColor.
Also, remove crash sections for now.

Reply:
done